### PR TITLE
Add how to execute as non root to README.md

### PR DESCRIPTION
**Pull request by cawamata**
https://github.com/soundcloud/ipmi_exporter/issues/32

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,32 @@ Make sure you have the following tools from the
  - `ipmi-dcmi`
  - `bmc-info`
 
+
+### How to run as non root
+
+When running this exporter as non root, you should the followings.
```
Comment (bitfehler) [inline README.md]: missing a "do" here (should _do_ the following)

File: README.md [hunk 2]
```diff
@@ -43,6 +43,32 @@ Make sure you have the following tools from the
...
+
+   1. add sudoers files to permit the above commands
```
Comment (bitfehler) [inline README.md]: maybe better "permit the _following_ commands"? The commands are not above that line.

File: README.md [hunk 3]
```diff
@@ -43,6 +43,32 @@ Make sure you have the following tools from the
...
+  3. create symlinks under user dir
+      ```bash
+      ln -s /home/ipmi-exporter/[script name] /home/ipmi-exporter/ipmimonitoring
+      ln -s /home/ipmi-exporter/[script name] /home/ipmi-exporter/ipmi-sensors
+      ln -s /home/ipmi-exporter/[script name] /home/ipmi-exporter/ipmi-dcmi
+      ln -s /home/ipmi-exporter/[script name] /home/ipmi-exporter/bmc-info
+      ````
+  4. execute ipmi-exporter with the option
+      ```
+      --freeipmi.path=/home/ipmi-exporter
```
Comment (bitfehler) [inline README.md]: Maybe make this inline code (with single backtick), doesn't need to take up that much space.

**Comment by cawamata** [comment]
@bitfehler 
Thank you for review. (and sorry of these easy miss)
I updated, can you check again?

**Comment by bitfehler** [comment]
Thanks a bunch!

**Pull request merged**